decoders/connector/lansitec/tracking-label/v1.0.0/payload.js

```diff
@@ -0,0 +1,215 @@
+// Tracking Label
```

Comment:
New decoders that are added must be made in TypeScript and must have a unit test.